<launch>
  <node name="stereo_cam_node" pkg="usb_cam" type="stereo_cam_node" output="screen" >
    <param name="video_device" value="cam" /> <!--  'cam' for using your stereo camera (eg ZED) OR 'video' for video file stored at location video_dir specified below -->
    <param name="source" value="/dev/video0" />
    <param name="video_dir" value="/home/interface/softwares/tensorflow/Tensorflow_dataset/videos/output_show.mp4" />   <!--   back_red_wait_1   splice_1    video1_p_r video1_person_r splice_final   -->
    <param name="image_width" value="640" /> <!--1344, 2560, 3840, 4416 -->
    <param name="image_height" value="480" /> <!--376,720,1080,1242 -->
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="left_camera_info_url" value="file:///home/interface/catkin_ws/src/usb_cam/calibration/left_1280_720.yaml"/>
    <param name="right_camera_info_url" value="file:///home/interface/catkin_ws/src/usb_cam/calibration/right_1280_720.yaml"/>
    <param name="framerate" value="40" />
  </node>
  <!-- node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node -->
</launch>
